app/serializers/rest/account_serializer.rb

```diff
@@ -8,7 +8,8 @@ class REST::AccountSerializer < ActiveModel::Serializer
 
   attributes :id, :username, :acct, :display_name, :locked, :bot, :discoverable, :indexable, :group, :created_at,
              :note, :url, :uri, :avatar, :avatar_static, :header, :header_static,
-             :followers_count, :following_count, :statuses_count, :last_status_at, :hide_collections
+             :followers_count, :following_count, :statuses_count, :last_status_at, :hide_collections,
+             :avatar_description, :header_description
```

Comment:
You might want to do:

```
def avatar_description
  object.unavailable? ? "" : object.avatar_description
end
```

Basically, if the account has been disabled in some way, then there will be the default avatar / header returned, not the header the user set.